docs/sources/mimir/operators-guide/mimir-runbooks/_index.md

```diff
@@ -145,6 +146,7 @@ How to **fix** it:
 
 1. Ensure shuffle-sharding is enabled in the Mimir cluster
 1. Assuming shuffle-sharding is enabled, scaling up ingesters will lower the number of tenants per ingester. However, the effect of this change will be visible only after `-blocks-storage.tsdb.close-idle-tsdb-timeout` period so you may have to temporarily increase the limit
+1. For small tenants, default allocations for TSDB stripes and chunk writer buffers may be oversized. If a cell contains only small tenants, these allocation sizes can be reduced to allow raising the `max_tenants` limit without causing greatly increased per-tenant memory consumption.
```

Comment:
I think the following phrasing makes clearer the alternative strategy of increasing the `max tenants` limit, and how one might avoid excessive memory resulting usage:
```suggestion
1. If the cell's number of series per ingester is below our target, we may want to increase the `max tenants` limit instead of adding more ingesters (which would make the cell over-provisioned in effect). If memory usage is high (due to the number of tenants), we might want to bring it down by reducing TSDB stripe size (`-blocks-storage.tsdb.stripe-size`) and chunk writer buffers (`-blocks-storage.tsdb.head-chunks-write-buffer-size-bytes`). The latter will have a negative performance impacts for large tenants, but for smaller tenants it should be OK. See [example PR](https://github.com/grafana/deployment_tools/pull/64936).
```